views/layouts/application.ejs

```diff
@@ -22,6 +22,7 @@
         integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
         crossorigin="anonymous"></script>
     <script type="application/javascript" src="/javascripts/application.js?1"></script>
```

Comment:
You probably want to bump the version for application.js here too

Reply:
Yup, done, spaced out on that. I'll try to come up with something that automates cache breaking, it's easy to forget to do.